scriptmodules/emulators/uae4arm.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie.
+# 
+# (c) Copyright 2012-2015  Florian Müller (contact@petrockblock.com)
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/petrockblog/RetroPie-Setup/master/LICENSE.md.
+#
+
+rp_module_id="uae4arm"
+rp_module_desc="Amiga emulator RPI2 with JIT support"
+rp_module_menus="4+"
+
+function depends_uae4arm() {
+    getDepends libsdl1.2-dev libsdl-mixer1.2-dev libsdl-image1.2-dev libsdl-gfx1.2-dev libsdl-ttf2.0-dev libguichan-dev libguichan-0.8.1-1 libguichan-sdl-0.8.1-1
+}
+
+function sources_uae4arm() {
+    gitPullOrClone "$md_build" https://github.com/Chips-fr/uae4arm-rpi/
+}
+
+function build_uae4arm() {
+    make
+    md_ret_require="$md_build/uae4arm"
+
+}
+function install_uae4arm() {
+    cp data "$md_inst" -r
+    md_ret_files=(
+        'uae4arm'
+  #      'uae4all'
+  #      'Readme.txt'
+    )
+}
+
+function configure_uae4arm() {
+    mkRomDir "amiga"
+
+    mkdir -p "$md_inst/conf"
+   #   echo "path=$romdir/amiga" >"$md_inst/conf/adfdir.conf"
+    chown -R $user:$user "$md_inst/conf"
+
+    mkdir -p "$md_inst/kickstarts"
+    chown -R $user:$user "md_inst/conf"
+    
```

Comment:
This should be $md_inst btw. Don't worry though, rather than merging the pull request, I will apply it manually with a couple of changes. thanks.